lib/Minz/ModelPdo.php

```diff
@@ -28,6 +28,9 @@ public function __construct($currentUser = null, $currentPdo = null) {
 		if ($currentUser === null) {
 			$currentUser = Minz_Session::param('currentUser');
 		}
+		if ($currentUser == '') {
+			throw new Minz_PDOConnectionException('Current user must not be empty!', '', Minz_Exception::ERROR);
```

Comment:
Another barrier to prevent logic from running without a proper user. The other bugs fixed in the PR allowed coming here with a blank user

Reply:
This breaks cli (cf. #2644 ) Removing this two lines is enough to make cli works again.

Reply:
@X-dark Thanks for the heads up

Reply:
Fixed by https://github.com/FreshRSS/FreshRSS/pull/2646